extensions/wrapper/client-example/build.gradle.kts

```diff
@@ -1,6 +1,6 @@
 plugins {
     java
-    id("io.quarkus") version "3.0.0.Final"
```

Comment:
Is there a reason why one downgrade the version of io.quarkus?

Reply:
Yes, the versions are supposed to be compatible. Since this is an example project I removed externalizing the version into gradle.properties, and I suspect the plugin version was bumped by dependabot.